server/src/repositories/config.repository.ts

```diff
@@ -10,6 +10,9 @@ export class ConfigRepository implements IConfigRepository {
         skipMigrations: process.env.DB_SKIP_MIGRATIONS === 'true',
         vectorExtension: getVectorExtension(),
       },
+      storage: {
+        ignoreMountCheckErrors: (process.env.IMMICH_IGNORE_MOUNT_CHECK_ERRORS ?? 'true') === 'true',
```

Comment:
Doesn't this default to disabling the checks?

Reply:
Yeah lol